android/build.gradle

```diff
@@ -46,10 +51,10 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
+    implementation "androidx.appcompat:appcompat:$androidxAppCompatVersion"
     testImplementation "junit:junit:$junitVersion"
     androidTestImplementation "androidx.test.ext:junit:$androidxJunitVersion"
     androidTestImplementation "androidx.test.espresso:espresso-core:$androidxEspressoCoreVersion"
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'io.intercom.android:intercom-sdk:8.+'
+    implementation 'io.intercom.android:intercom-sdk:9.+'
```

Comment:
why not use sdk 10?

Reply:
At first I did not because of concerns around breaking changes. I read through the changelog though and believe we should be good. The only difference was the change from `hideMessenger` to `hideIntercom` but now the behavior matches that of iOS.

Reply:
Yes, that makes sense.